src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/reductions/UpdateReductionServiceTest.kt

```diff
@@ -0,0 +1,70 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.integration.reductions
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.data.repository.findByIdOrNull
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.repository.LduRepository
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.repository.RegionRepository
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.LduEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.OffenderManagerEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.ReductionEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.ReductionStatus
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.RegionEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.TeamEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.WMTWorkloadOwnerEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.reduction.GetReductionService
+import uk.gov.justice.digital.hmpps.hmppsworkload.service.reduction.UpdateReductionService
+import java.math.BigDecimal
+import java.time.ZonedDateTime
+
+class UpdateReductionServiceTest : IntegrationTestBase() {
+  @Autowired
+  lateinit var regionRepository: RegionRepository
+
+  @Autowired
+  lateinit var lduRepository: LduRepository
+
+  @Autowired
+  lateinit var getReductionService: GetReductionService
+
+  @Autowired
+  lateinit var updateReductionService: UpdateReductionService
+
+  lateinit var workloadOwner: WMTWorkloadOwnerEntity
+
+  @BeforeEach
+  fun setupReductionTestData() {
+    val region = regionRepository.save(RegionEntity(code = "REGION2", description = "Region 2"))
+    val pdu = lduRepository.save(LduEntity(code = "LDU2", description = "Local Delivery Unit 2(Actually a Probation Delivery Unit)", region = region))
```

Comment:
is it worth renaming this repoository?

Reply:
doing work in this area, will do that in that PR